assets/training/finetune_acft_image/components/model_import/mmd_od_is/spec.yaml

```diff
@@ -1,14 +1,14 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
 type: command
 
-version: 0.0.13
+version: 0.0.14
```

Comment:
Do we need to bump up version for this? We are not changing anything for MMD model import?

Reply:
The envrionment version is getting updated. To use the same env version in model import as well as finetune.